test/reverseregistrar_test.js

```diff
@@ -74,4 +97,48 @@ describe('ReverseRegistrar', function() {
             .then(txHash => ens.ownerAsync(node))
             .then(result => assert.equal(result, accounts[1]));
     });
+
+    it('allows an account to specify resolver', function() {
+        this.slow(150);
+        return registrar.claimWithResolverAsync(accounts[1], accounts[2], {from: accounts[0]})
+            .then(txHash => ens.ownerAsync(node))
+            .then(result => assert.equal(result, accounts[1]))
+            .then(result => ens.resolverAsync(node))
+            .then(result => assert.equal(result, accounts[2]));
+    });
+
+    it('does not overwrite resolver if not specified', function() {
+        this.slow(150);
+        return registrar.claimWithResolverAsync(accounts[1], accounts[2], {from: accounts[0]})
+            .then(txHash => registrar.claimAsync(accounts[3], {from: accounts[0]}))
+            .then(txHash => ens.ownerAsync(node))
+            .then(result => assert.equal(result, accounts[3]))
+            .then(result => ens.resolverAsync(node))
+            .then(result => assert.equal(result, accounts[2]));
+    })
+
+    it('sets name records', function() {
+        this.slow(300);
+        return registrar.setNameAsync('testname', {from: accounts[0]})
```

Comment:
Could be the case - see [successful Travis build from my repo](https://travis-ci.org/veox/ens/builds/235608950), with an [arbitrarily high gas specified](https://github.com/veox/ens/commit/8e56257eb309ca84da40181081f055b5e487bf24).